addons/website_forum/views/website_forum.xml

```diff
@@ -142,129 +127,205 @@
-                    <i class="ml-4 fa fa-comments text-primary"/>
-                    <a t-attf-href="/forum/#{ slug(question.forum_id) }/question/#{ slug(question) }">
-                        <t t-esc="question.child_count"/>
-                        <t t-if="question.child_count&lt;=1">Answer</t>
-                        <t t-else="">Answers</t>
-                    </a>
-                    <a t-attf-href="/forum/#{ slug(question.forum_id) }/question/#{ slug(question) }">
-                        <span class="ml-4 mr-1 fa fa-star"></span><span t-field="question.favourite_count"/> <t t-if="question.views&lt;=1">Favorite</t><t t-else="">Favorites</t>
-                    </a>
-                    <span t-if="question.state == 'flagged'" class="text-black">
-                        <i class="fa fa-flag ml-4 mr4"/>
-                        Flagged
-                    </span>
-                </small>
-            </div>
+
+    <div t-attf-class="o_wforum_index_entry_tags mb-1" t-if="len(question.tag_ids) > 0">
+        <t t-foreach="question.tag_ids" t-as="question_tag">
+
+            <!-- Toggle Tags on click -->
+            <t t-if="tag and tag.name == question_tag.name" t-set="click_action"
+                t-value="'/forum/' + slug(question_tag.forum_id) + '?' + keep_query( 'search', 'sorting', 'my')"/>
+            <t t-else="" t-set="click_action"
+                t-value="'/forum/' + slug(question_tag.forum_id) + '/tag/' + slug(question_tag) + '/questions?' + keep_query( 'search', 'sorting', 'my', filters='tag')"/>
+
+            <a t-att-href="click_action"
+                t-attf-class="badge #{tag and tag.name == question_tag.name and 'badge-secondary' or 'border text-600 badge-light'} #{'ml-lg-1 mt-lg-1' if not (question_tag_first) else ''}"
+                t-field="question_tag.name"/>
+        </t>
+    </div>
+
+    <div class="d-flex align-items-center justify-content-between justify-content-sm-start small text-muted">
+        <div>
+            <t t-call="website_forum.vote">
+                <t t-set="post" t-value="question"/>
+            </t>
+            <span t-field="question.write_date" t-options='{"format": "d MMMM y"}'/>, by <a t-attf-href="/forum/#{slug(question.forum_id)}/user/#{question.create_uid.id}" t-field="question.create_uid" class="font-weight-bold" t-options='{"widget": "contact", "fields": ["name"]}'/>
+        </div>
+        <div>
+            <span class="mx-1 d-none d-sm-inline">|</span>
+            <a t-if="len(question.child_ids)&gt;0" class="font-weight-bold" t-attf-href="/forum/#{ slug(question.forum_id) }/question/#{ slug(question) }">
+                <t t-esc="question.child_count"/>
+                <t t-if="question.child_count == 1">Answer</t>
+                <t t-else="">Answers</t>
+            </a>
+            <span t-else="">
+                0 Answers
```

Comment:
I'm not english expert... Are you sure about the 0 and plural form.. ?

0 Answers

Reply:
Seems ok > https://hinative.com/en-US/questions/5411